semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/operatorextension/nodes/Arithmetic.scala

```diff
@@ -0,0 +1,5 @@
+package io.shiftleft.semanticcpg.language.operatorextension.nodes
+
+import io.shiftleft.codepropertygraph.generated.nodes
+
+class Arithmetic(val call: nodes.Call) extends AnyRef {}
```

Comment:
Ditto - is there any reason to have an empty class? The `extends AnyRef` can also be dropped as it is implicit.